eve/workers/unit_and_feature_tests/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM buildpack-deps:jessie-curl
 ENV LANG C.UTF-8
 COPY utapi_packages.list buildbot_worker_packages.list /tmp/
 
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+RUN curl -sL https://deb.nodesource.com/setup_11.x | bash - \
```

Comment:
does 11.x install node11? I do not think that version is part of LTS, I think they do every other version. Might want to use 12 or an exact version of 10 that is later. Could totally be wrong here tho.

Reply:
Yeah, I'm playing around with it right now. I need v10.19.0 but there's no PPAs for specific minor versions :[

Reply:
lame, I would expect 10.x to install the latest minor+patch version as well.

Reply:
yeah, i was surprised by that too. but it installs 10...16? or 14. either way, super lame